com.avaloq.tools.ddk.xtext.test.core/src/com/avaloq/tools/ddk/xtext/test/scoping/AbstractScopingTest.java

```diff
@@ -136,6 +168,17 @@ protected void beforeAllTests() {
     getTestInformation().putTestObject(Iterable.class, allContents);
   }
 
+  /**
+   * Test linking.
+   */
+  @Override
+  protected void afterEachTest() {
+    expectedLinks.forEach(ExpectedLink::test);

```

Comment:
I'd rather we test the links in some other method.

Reply:
Done.